elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSchemaFactory.java

```diff
@@ -131,10 +135,20 @@ public ElasticsearchSchemaFactory() {
         ("Both 'coordinates' and 'hosts' is missing in configuration. Provide one of them.");
       }
       final String pathPrefix = (String) map.get("pathPrefix");
+
+      // Add or Disable SSL Verification
+      boolean disableSSLVerification;
+      if (map.containsKey("disableSSLVerification")) {
+        disableSSLVerification = (Boolean) map.get("disableSSLVerification");
```

Comment:
cast string directly to boolean ?

Reply:
Fixed.  I broke this into two steps to avoid a possible NPE.